app/controllers/stories_controller.rb

```diff
@@ -2,7 +2,8 @@
 
 class StoriesController < ApplicationController
   def index
-    @unread_stories = authorization.scope(StoryRepository.unread)
+    order = current_user.stories_order
+    @unread_stories = authorization.scope(StoryRepository.unread(order:))
```

Comment:
I'm not super familiar with fever api, but should the queries in there be updated to match as well? Or do we expect the client to handle it instead?

Reply:
I feel like that should be the clients responsibility. Unfortunately the original spec seems gone, but I doubt it specified any contract for this at all given the overall variance in the client implementations.

Given the fever API allows partial syncs with `since_id`, I can't imagine server side sorting to work reliably anyhow, imagine a sync history where the order is flipped between two partial syncs.

Reply:
Yep, looking at [archive.org](https://web.archive.org/web/20221221112459/https://feedafever.com/api), there aren't even any response formats described, let alone semantics like item order.